src/sda/Application.java

```diff
@@ -1,14 +1,18 @@
 package sda;
 
 import sda.calculator.Calculator;
+import sda.calculator.RandomGenerator;
 
 public class Application {
     public static void main(String[] args) {
         Calculator calculator = new Calculator();
-        System.out.println(calculator.sum(5, 10));
-        System.out.println(calculator.subtract(5, 10));
-        System.out.println(calculator.multiply(5, 10));
-        System.out.println(calculator.divide(5, 10));
-        System.out.println(calculator.divide(5, 0));
+        RandomGenerator randomGenerator = new RandomGenerator();
+
+        System.out.println(calculator.sum(randomGenerator.getSmallNumber(), randomGenerator.getBigNumber()));
+        System.out.println(calculator.subtract(randomGenerator.getSmallNumber(), randomGenerator.getBigNumber()));
+        System.out.println(calculator.multiply(randomGenerator.getSmallNumber(), randomGenerator.getBigNumber()));
+        System.out.println(calculator.divide(randomGenerator.getSmallNumber(), randomGenerator.getBigNumber()));
```

Comment:
`randomGenerator.getSmallNumber()` and `randomGenerator.getBigNumber()` could be extracted to their own functions. Don't repeat yourself for life 💯

Reply:
ok my bad they are already extracted, I don't know Java :D